docs/semgrep-code/semgrep-pro-engine-intro.md

```diff
@@ -179,20 +179,20 @@ rules:
+If a scan uses more than 5&nbsp;GB of memory during pre-processing, the scan uses Semgrep OSS Engine to ensure lower memory consumption. Similarly, if the Semgrep Pro Engine scan doesn't complete after 3 hours, the Pro Engine times out and Semgrep OSS rescans the repository. Typically, this is because the repository is very large.
 
 If 1-2 repositories cause CI scan issues, modify your config file to use `semgrep ci --oss-only`. This overrides the Semgrep CI settings for these repositories, and always runs these scans with Semgrep OSS. 
 
 If many repositories cause scan issues:
 1. Disable the <i class="fa-solid fa-toggle-large-on"></i> **Semgrep Pro Engine beta** toggle in the **[Settings](https://semgrep.dev/orgs/-/settings)** page of your organization.
-1. Contact r2c in the <a href="https://go.semgrep.dev/slack">Semgrep Community Slack</a> so we can help you to fix the issue and create a plan for your organization.
+1. Contact Semgrep in the <a href="https://go.semgrep.dev/slack">Semgrep Community Slack</a> so we can help you to fix the issue and create a plan for your organization.
 
 ### Difference between Semgrep Pro Engine and join mode
 
-Semgrep Pro Engine is different from [join mode](/writing-rules/experiments/join-mode/overview/), which also allows you to perform interfile analyses by letting you join on the metavariable matches in separate rules. Join mode is an experimental feature which is not developed or actively maintained. You may encounter many issues while using join mode.
+Semgrep Pro Engine is different from [join mode](/writing-rules/experiments/join-mode/overview/), which also allows you to perform interfile analyses by letting you join on the metavariable matches in separate rules. Join mode is an experimental feature which is not actively developed or maintained. You may encounter many issues while using join mode.
 
 ### Future development of Semgrep Pro Engine
 
-We’re excited to hear what’s on your mind. As users explore the limits of Semgrep Pro Engine, we want to know what they’re failing to express. We believe that this deeper analysis helps users find more vulnerabilities, build trust with developers, and enforce code standards quickly. Let us know what you think about the results in the <a href="https://go.semgrep.dev/slack">Semgrep Community Slack</a>.
+We’re excited to hear what’s on your mind. As you explore Semgrep Pro Engine, we want to know what you'd like to be able to capture with Pro Engine. We believe that this deeper analysis helps users find more vulnerabilities, build trust with developers, and enforce code standards quickly. Let us know what you think about the results in the <a href="https://go.semgrep.dev/slack">Semgrep Community Slack</a>.
```

Comment:
I'd love to hear other thoughts on how to structure this paragraph. For example, maybe we want to move the "We believe" to the beginning, or even remove it? It doesn't seem directly relevant to the topic "Future development".

Reply:
Hi @armchairlinguist, thank you for looking into this! My thoughts are:

* I think what this heading is really asking for is feedback and is building a case for transparency (hence the "we believe..." line.) Maybe we could use a header such as "Developing Semgrep Pro Engine through feedback" or simply "Feedback for Semgrep Pro Engine"
* Speaking from a "strict" TW standpoint, marketing copy ("We believe X") is used in marketing collateral. I personally am not very strict when a doc is created around launch time (as this one was), as the only collateral to send to users **is** the docs and we try to be more approachable. As the product matures, the content usually evolves organically to be more of an "official handbook". @milanwilliams What do you think about removing it?
* Optional change: We could identify "we" at first use, such as "The team at Semgrep is excited to hear what's on your mind." (This is in line with the [Google Documentation Style Guide](https://developers.google.com/style/pronouns).)

Thank you for opening this discussion.

Reply:
For my 2c - I like the update you made @armchairlinguist! And agree we should update the heading to make it clearer, I liked `Feedback for Semgrep Pro Engine` feels short & to the point :)